src/frequenz/sdk/microgrid/_battery/_status.py

```diff
@@ -137,18 +145,17 @@ class BatteryStatusTracker(AsyncConstructible):
         InverterComponentState.COMPONENT_STATE_STANDBY,
     }
 
-    # Instance attributes
-    _battery_id: int
-    _max_data_age: float
-    _last_status: BatteryStatus
-    _blocking_status: _BlockingStatus
-    _battery: _ComponentData[BatteryData]
-    _inverter: _ComponentData[InverterData]
-
-    @classmethod
-    async def async_new(
-        cls, battery_id: int, max_data_age_sec: float, max_blocking_duration_sec: float
-    ) -> BatteryStatusTracker:
+    # Instance attributes defined outside __init__
+    _select: Optional[Select]
```

Comment:
You don't need this line.  You can just do this inside the `__init__` method, like you're already doing for other variables:

``` python
self._select: Optional[Select] = None
```

Reply:
Done